adaptor/docs/ReleaseProcess.html

```diff
@@ -193,17 +202,45 @@
           </code>
         </pre>
 
+        <p><strong>Note:</strong> Keep this clone until a subsequent
+        RC is built, or the release is published.</p>
+
         <p><strong>Note:</strong> It is possible to reuse this directory
         for subsequent RC builds, by leaving out some steps and altering
         others, but that is not recommended or documented.</p>
 
         <h3> <a id="Release"></a> Release the library or adaptor </h3>
+        <p>Push the branch and tag to GitHub.</p>
         <pre>
           <code>
+            # Release branch names start with "v" and end in "x".
+            BRANCH=v4.1.x
+            VERSION=4.1.2
+
             # Push the branch and tag.
-            git push $BRANCH v$VERSION
+            git push -u origin $BRANCH v$VERSION
           </code>
         </pre>
+        <p>Add a new release on GitHub, using the new tag. Use a release
```

Comment:
Maybe mention where to start this process: "from the releases tab, click 'draft a new release'" (or something, for those who haven't ever created a release). The UI is kind of busy and it's sometimes hard to identify where to start different things.